src/GDGUkraine/lib/utils/mail.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import base64
+import json
+
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+import html2text
+from blueberrypy.template_engine import get_template
+
+logger = logging.getLogger(__name__)
+
+
+def gmail_send(oauth2session, message, sbj, to_email,
```

Comment:
I've dropped `oauth2session` param. See 9f9fb7447c16a6f346026acf4534a0b95df88818.
Plz port this commit.